app/cypress/integration/cif/project-revision/create-first-project-revision.spec.js

```diff
@@ -119,7 +119,7 @@ describe("when creating a project, the project page", () => {
     cy.happoAndAxe("Emission Intensity Form", "filled", "main");
     cy.findByText(/Submit Emissions Intensity Report/).click();
 
-    //add quarterly reports
+    cy.log("Add Quarterly Reports");
```

Comment:
Log statement left behind?

Reply:
It was, they're actually quite nice to have in there for local testing, but it's just adding time to include it for the CI runs. Pushed the switch back to a comment